src/base/bittorrent/session.cpp

```diff
@@ -2416,22 +2430,42 @@ QStringList Session::getListeningIPs() const
 
     const QString ifaceName = networkInterface();
     const QString ifaceAddr = networkInterfaceAddress();
-    const bool listenIPv6 = isIPv6Enabled();
+    const QHostAddress configuredAddr(ifaceAddr);
+    const bool allIPv4 = (ifaceAddr == QLatin1Char('4')); // Means All IPv4 addresses
```

Comment:
Maybe use some more IP-specific placeholders (e.g. `0.0.0.0` and `::0`) instead? It appears in configure file and can confuse the readers.

Reply:
Interesting. I'll take a look into the rest of the code and if it doesn't break something, I'll change it to your suggestion.